spec/features/create_work_spec.rb

```diff
@@ -6,7 +6,7 @@
   let(:file2) { File.open(fixture_path + '/image.jp2') }
   let!(:uploaded_file1) { Hyrax::UploadedFile.create(file: file1, user: user) }
   let!(:uploaded_file2) { Hyrax::UploadedFile.create(file: file2, user: user) }
-  let(:permission_template) { create(:permission_template, source_id: 'admin_set/default', with_admin_set: true, with_active_workflow: true) }
+  let(:permission_template) { create(:permission_template, with_admin_set: true, with_active_workflow: true) }
```

Comment:
The source_id was never referenced explicitly so removing it allows it to be any admin set.